src/index.ts

```diff
@@ -85,6 +86,7 @@ program
     .option('-r, --replace', 'Replace the input project. Cannot be used with --output')
     .option('-b, --editor-bundle <path>', `The editor bundle that was generated by the Wonderland Editor for the input project when building (default: "<project-path directory>/${EDITOR_BUNDLE_DEFAULT_ROOT}/<project-name>/${EDITOR_BUNDLE_DEFAULT_FILE}")`)
     .option('-e, --editor-bundle-extra <path>', `Add extra definitions to the editor bundle via a JS script (default: "${EDITOR_BUNDLE_EXTRA_DEFAULT}")`)
+    .option('-t, --throw', `Enable throwing errors on unexpected conditions which can be normally ignored or skipped`)
```

Comment:
this is a safety feature. you should have it be the opposite to turn it off instead (e.g. `--no-throw`), this way:
- people who don't know what they're doing keep using the safety feature
- the default behaviour is not changed